src/replica/replica_stub.cpp

```diff
@@ -389,9 +443,36 @@ void replica_stub::initialize(const replication_options &opts, bool clear /* = f
         }
     }
 
+    std::string encryption_key;
+    std::string iv;
+    std::string key_version;
+    std::string server_key;
+    // get and store Encrypted Encryption Key(eek),Initialization Vector(iv),Key Version from kms
```

Comment:
```suggestion
    // Get and store eek (a.k.a encrypted encryption key), iv (a.k.a. initialization vector), key version from KMS.
```

Or add the comments to the replica_kms_info struct defination if using its object here.